e2e-tests/cypress/e2e/journeys/insurance/your-buyer/change-your-answers/change-your-answers-change-company-or-organisation.spec.js

```diff
@@ -0,0 +1,269 @@
+import partials from '../../../../partials';
+import { FIELD_IDS, ROUTES, WEBSITE_EXAMPLES } from '../../../../../../constants';
+import { companyOrOrganisationPage, checkYourAnswersPage } from '../../../../pages/insurance/your-buyer';
+import { submitButton } from '../../../../pages/shared';
+import { YOUR_BUYER_FIELDS as FIELDS } from '../../../../../../content-strings/fields/insurance/your-buyer';
+import application from '../../../../../fixtures/application';
+
+const {
+  INSURANCE: {
+    YOUR_BUYER: {
+      COMPANY_OR_ORGANISATION: {
+        NAME,
+        ADDRESS,
+        COUNTRY,
+        REGISTRATION_NUMBER,
+        WEBSITE,
+        FIRST_NAME,
+        LAST_NAME,
+        POSITION,
+        EMAIL,
+        CAN_CONTACT_BUYER,
+      },
+    },
+  },
+} = FIELD_IDS;
+
+const {
+  YOUR_BUYER: {
+    COMPANY_OR_ORGANISATION_CHANGE,
+    CHECK_YOUR_ANSWERS,
+  },
+  START,
+} = ROUTES.INSURANCE;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.buyer;
+
+const { summaryList } = checkYourAnswersPage;
+
+context('Insurance - Your buyer - Change your answers - Company or organisation - As an exporter, I want to change my answers to the company or organisation section', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.navigateToUrl(START);
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    task.link().click();
+
+    cy.completeAndSubmitCompanyOrOrganisationForm();
+    cy.completeAndSubmitWorkingWithBuyerForm();
+
+    cy.getReferenceNumber().then((id) => {
+      referenceNumber = id;
+    });
```

Comment:
Need to update this after other PR merged, to use sign in command, makes it simpler as well:

```js
before(() => {
  cy.completeSignInAndGoToApplication().then((refNumber) => {
    referenceNumber = refNumber;

    // go to the page we want to test.
    taskList.prepareApplication.tasks.policyTypeAndExports.link().click();
```

Reply:
Sorry just realised waiting for this to be merged first https://github.com/UK-Export-Finance/exip/pull/354